_stbt/tv_driver.py

```diff
@@ -163,9 +163,10 @@ def add_argparse_argument(argparser):
 
 def create_from_args(args, video_generator):
     desc = args.tv_driver
-    video_server = _HTTPVideoServer(
-        video_generator,
-        video_format=get_config('camera', 'video_format'))
+    if desc.startswith('fake:') or desc == 'manual':
+        video_server = _HTTPVideoServer(
+            video_generator,
+            video_format=get_config('camera', 'video_format'))
```

Comment:
My biggest concern with the patch is that it adds an additional `if` statement that must be kept in-sync with the if statement below.  Perhaps a better approach would be to move the creation of `video_server` below into the existing `if`, `elif` ladder.

Perhaps we can use Python's closures to avoid duplication and additional conditional logic.  Something like:

    def make_video_server():
        return _HTTPVideoServer(
            video_generator,
            video_format=get_config('camera', 'video_format'))

and then call that when instantiating `_FakeTvDriver` or `_ManualTvDriver`.